plyer/facades/sms.py

```diff
@@ -1,3 +1,22 @@
+'''
+Sms
+====
+
+The :class:`Sms` provides access to sending Sms from your device.
+
+Simple Examples
+---------------
+
+To send sms::
+
```

Comment:
would be nice if you could mention he permissions needed for every such example too.